frontend/front/package.json

```diff
@@ -16,6 +16,7 @@
     "animate.css": "^4.1.1",
     "classnames": "^2.3.2",
     "clsx": "^1.2.1",
+    "dotenv": "^16.0.3",
```

Comment:
Is dotenv actually getting used anywhere here? Regardless I think it's good to have.